source/BoardingPanel.cpp

```diff
@@ -108,7 +108,7 @@ BoardingPanel::BoardingPanel(PlayerInfo &player, const shared_ptr<Ship> &victim)
 	}
 
 	// Some "ships" do not represent something the player could actually pilot.
-	if(!victim->IsCapturable())
+	if(!victim->IsCapturable() && !overrideCapture)
```

Comment:
`victim->IsCapturable() || overrideCapture` is used in a couple of places. I think you should that as a variable instead of `overrideCapture`, because the latter is never used without `IsCapturable()` anyways.